buildSrc/src/main/kotlin/Config.kt

```diff
@@ -57,6 +57,7 @@ object Versions {
     const val androidxTest = "1.3.0" // https://maven.google.com/web/index.html#androidx.test:rules
     const val autoService = "1.0-rc6"
     const val detektPlugin = "1.14.1"
+    const val dokka = "1.4.20"
```

Comment:
```suggestion
    const val dokka = "1.4.20" // https://github.com/Kotlin/dokka
```

Just adding an easy reference, so it is quick to lookup the source and find out if the version changed